client/composeApp/src/commonMain/kotlin/io/aoriani/ecomm/data/model/ProductBasic.kt

```diff
@@ -0,0 +1,9 @@
+package io.aoriani.ecomm.data.model
+
+interface ProductBasic {
```

Comment:
Why didn't you suggest adding documentation to this interface.  I said that to your review guidelines/